README.md

```diff
@@ -19,7 +19,7 @@ Add the following dependency to your build.gradle:
 
 `implementation group: 'com.cloudinary', name: 'cloudinary-android', version: '1.30.0'`
 ## Manual Setup ######################################################################
-Download cloudinary-android-1.30.0.jar from [here](http://central.maven.org/maven2/com/cloudinary/cloudinary-android/1.30.0/cloudinary-android-1.30.0.aar) and cloudinary-core-1.26.0.jar from [here](http://central.maven.org/maven2/com/cloudinary/cloudinary-core/1.26.0/cloudinary-core-1.26.0.jar) and put them in your libs folder.
+Download the latest Cloudinary Android SDK from [here](https://repo1.maven.org/maven2/com/cloudinary/cloudinary-android-core/1.30.0/cloudinary-android-core-1.30.0.aar) and the latest core version from [here](https://repo1.maven.org/maven2/com/cloudinary/cloudinary-core/1.29.0/cloudinary-core-1.29.0.jar) and put them in your libs folder.
```

Comment:
@adimiz1 , although already merged, maybe we should put a link to the listing page of all versions rather than to a specific version (that we'll need to updated on every release)

Reply:
Agreed, will fix.